Yoast/YoastSEO.js
# Add internal linking assessment

## Summary

This PR can be summarized in the following changelog entry:

## Relevant technical choices:

* Renamed textlinks to `outboundLinks` because that's what is does

## Test instructions

This PR can be tested by following these steps:

* Add some internal links and see the results

Fixes #1136
